apps/storefront/src/components/layout/B3Mainheader.tsx

```diff
@@ -45,7 +48,7 @@ export default function B3Mainheader({ title }: { title: string }) {
       return;
     }
 
-    window.location.href = CART_URL;
+    window.location.href = window.B3.setting.platform === 'catalyst' ? '/cart' : CART_URL;
```

Comment:
@bc-victor - shouldn't we provide a way to customize this for other platforms?
I think we should change the `CART_URL` variable dynamically based on `window.B3.setting.cart_url`.

The fallback value of `CART_URL` when `window.B3.setting.cart_url` is not defined can be `/cart.php` when platform is `bigcommerce` vs `/cart` for other platforms.